import/popups/ContextMenu.qml

```diff
@@ -211,6 +211,17 @@ ContextMenuInterface {
                 onClicked: root._hide()
             }
 
+            MenuItem {
+                visible: root.isImage && root.url
+                //: Copy image address to clipboard from context menu
+                //% "Copy image address"
+                text: qsTrId("sailfish_components_webview_popups-me-copy_address_to_clipboard")
+                onClicked: {
+                    root._hide()
+                    Clipboard.text = root.url
```

Comment:
This should be ShareAction to share an image. Similar to share option in the browser menu to share a url.